Loop/Extensions/NSUserDefaults.swift

```diff
@@ -80,7 +80,7 @@ extension UserDefaults {
                     return .enlite
                 }
 
-                if let transmitterID = string(forKey: "com.loudnate.Naterade.TransmitterID"), transmitterID.characters.count == 6 {
+                if let transmitterID = string(forKey: "com.loudnate.Naterade.TransmitterID"), transmitterID.count == 6 {
```

Comment:
this compatibility fix (and others) should probably be in its own PR to avoid merge conflicts, but I'm not going to be a stickler (maybe somebody else will be :-) )

Reply:
Agreed - it actually got in there by accident.  Wasn't working from the branch I thought I was...